akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.util
+
+import akka.annotation.InternalApi
+import akka.japi.function.{ Function ⇒ JFun, Function2 ⇒ JFun2 }
+import akka.japi.{ Pair ⇒ JPair }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object ConstantFun {
```

Comment:
Hm, I guess few people will play around their own version of Akka HTTP and even deploy their modified version on production. However, maybe you are right, we should take care of that situation.

Since Akka HTTP was just moved from Akka to a new repo, I think there might be more *INTERNAL APIs* in Akka are touched by Akka HTTP. I just did a very quick check, `OptionVal` in `akka.util` is also used in several places in Akka HTTP. If Akka HTTP should use their own version of those internal stuffs, should we create another issue to fix it together?